tests/integration-production.test.js

```diff
@@ -0,0 +1,41 @@
+/* eslint  jest/no-standalone-expect: 0 */
+const path = require('path');
+const fs = require('fs');
+
+describe(__filename, () => {
+  const DIST_DIR = path.join(__dirname, '..', 'dist');
+  // If the `SKIP_PRODUCTION_BUILD_TESTS` env variable is set (to `'1'`), we
+  // skip the test cases below, otherwise we run them (the default).
+  const maybeIt =
```

Comment:
This seems like a kind of convoluted way of skipping all of the tests in this file. I guess you are doing this to avoid an error/warning from Jest that there are no tests (if you were to just wrap them in an `if`)? Is there another way to avoid using an env variable like this? A way to just target specific tests in the `yarn` commands? I know Jest likes to run all tests by default. What if this `integration-production` test were in an entirely separate directory?

Reply:
I haven't found anything that would be better than that. Jest looks for `.test.js` files _everywhere_.

> Also, is this just a start, or all we are planning to do with these checks? The impression I got from issue #159 was that we wanted to test some of the logic inside the blog pages.